docs/rules/NoAutoTupling.md

```diff
@@ -4,27 +4,23 @@ id: NoAutoTupling
 title: NoAutoTupling
 ---
 
-> ⚠️ This rule does not work with Scala 2.13 since `-Yno-adapter-args` has been
-> removed.
-
 Adds explicit tuples around argument lists where auto-tupling is occurring.
 
 To use this rule:
 
-- enable `-Ywarn-adapted-args` (note, `-Yno-adapted-args` will fail compilation,
-  which prevents scalafix from running)
-- disable `-Xfatal-warnings`. Unfortunately, the Scala compiler does not support
-  finer grained control over the severity level per message kind. See
-  [scalameta/scalameta#924](https://github.com/scalameta/scalameta/issues/924)
```

Comment:
Did you remove that following https://github.com/scalacenter/sbt-scalafix/pull/196? It's specific to sbt-scalafix, but I guess it's not worth mentioning "just" for the other clients, so I am inclined to say that we should ignore mentions of `-Xfatal-warnings` considering sbt-scalafix is the main client.

Reply:
yes and I forgot about the other clients. I think the other client should mention it (or are already menting it)